### DISCUSSION TITLE
[Help]: Carousel when inside carousel touch-swipe problem.

### DISCUSSION THREAD

**Discussion by MoiseevT** [davidjerleke/embla-carousel#420] [Help] [answered]
Hi David, ty for your work.

I`m kinda new into React-development, can you guys point me into right direction? I`ve solved many cool tasks thanks to this library, but completely stuck at this one. I need to make carousel work inside carousel, and i actualy did it. But when i`m trying to interact with any layer of carousel, it triggers the other carousel aswell (so when i`m trying to slide one inslide, it slides outside one aswell)

![image](https://user-images.githubusercontent.com/110160624/213204010-c81d6560-bf8e-4abf-ae6a-bfda023b5672.png)

And i have zero clue how to prevent this. Can someone give me an advice pls?

**Comment by davidjerleke** [comment, marked as answer]
Hi @MoiseevT,

Thanks for your question. Do you mean like [this CodeSandbox](https://codesandbox.io/s/embla-carousel-react-github-issue-141-eg8b1)?

Best,
David

**Reply by MoiseevT** [reply]
Yep, TY very much. Have a nice day! : )
